app/Http/Controllers/BrandController.php

```diff
@@ -20,21 +20,21 @@ public function addBrand()
 
     public function store(BrandRequest $request)
     {
-        Brand::create($request->validated());
+        Brand::create($request->all());
```

Comment:
Why this change?

Reply:
rajansir a kidhu thu  [https://github.com/jayesh-parmar/inventory-management/pull/16#discussion_r1329569230](url)

Reply:
@rajan-parmar Please recreate the main issue as discussed

Reply:
@gauravmak I generate the issue at my side & mistake was `->ignore($this->product, 'id')` it should be `->ignore($this->product)`. same I explained to @jayesh-parmar over the call and he will make changes